contracts/cw20-merkle-airdrop/src/msg.rs

```diff
@@ -0,0 +1,170 @@
+use crate::ContractError;
+use cosmwasm_std::{from_slice, Binary, Uint128};
+use cw_utils::{Expiration, Scheduled};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct InstantiateMsg {
+    /// Owner if none set to info.sender.
+    pub owner: Option<String>,
```

Comment:
do we need it? can't we do just like with the credits contract that the config.owner is always assigned the info.sender value?

Reply:
That's also from original contract. I can remove it if you desire, but, again, @oopcode asked me to not introduce unnecessary patches.

Reply:
I don't really like when there is a code that's never meant to be used because it complicates code review very much for no reason actually. but okay, if you've been directly asked for it, that's kind of fine

Reply:
imo can be done in a separate PR. marking it with this msg to create a list of such complaints a bit later

Reply:
I have simplified this part, too.